test/packaging/windows/msi.tests.ps1

```diff
@@ -80,6 +80,33 @@ Describe -Name "Windows MSI" -Fixture {
         }
     }
 
+    Context "Upgrade code" {
+        BeforeAll {
+            $previewUpgladeCode = '39243d76-adaf-42b1-94fb-16ecf83237c8'
```

Comment:
If we add tests for Preview why not for Release?

If we add tests for x64 should we skip the tests on x86?